src/Gui/Forms/MainFormInteractor.cs

```diff
@@ -329,11 +329,57 @@ public void AddMetadataFile()
         /// <summary>
         /// Prompts the user for a script file and adds to the project.
         /// </summary>
-        public void AddScriptFile()
+        private void AddScriptFile()
         {
             var fileName = uiSvc.ShowOpenFileDialog(null);
             if (fileName == null)
                 return;
+            AddScriptFile(fileName);
+        }
+
+        /// <summary>
+        /// Prompts the user for a creating new script file and adds it to the
+        /// project.
+        /// </summary>
+        private void NewScriptFile()
```

Comment:
Naming: let's use `CreateScriptFile`, following the convention of using verbs in method names.

Reply:
Done in ff4f4c7